samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/readme.md

```diff
@@ -0,0 +1,162 @@
+# Get started sending and receiving messages from ServiceBus queues using QueueClient
+
+In order to run the sample in this directory, replace the following bracketed values in the `Program.cs` file.
+
+```csharp
+const string ServiceBusConnectionString = "{ServiceBus connection string}";
+const string QueueName = "{Queue path/name}";
+```
+
+Once you replace the above values run the following from a command prompt:
+   
+```
+dotnet restore
+dotnet build
+dotnet run
+```
+
+## The Sample Program
+To keep things reasonably simple, the sample program keeps send and receive code within a single hosting application.
+Typically in real world applications these roles are often spread across applications, services, or at least across 
+independently deployed and run tiers of applications or services. For clarity, the send and receive activities are kept as 
+separate methods as if they were different apps.
+
+For further information on how to create this sample on your own, follow the rest of the tutorial.
+
+## What will be accomplished
+In this tutorial, we will write a console application to send and receive messages to a ServiceBus queue using a QueueClient.
+QueueClient offers a simple API surface to send message(or messages in a batch) and offers a simple MessagePump model to receive messages.
+Once a message process handler is registered as shown below, the User code does not have to write explicit code to receive messages and 
+if configured using `MessageHandlerOptions`, does not have to write explicit code to renew message locks or complete messages or improve 
+the degree of concurrency of message processing. Hence the queueClient can be used in scenarios where the User wants to get started 
+quickly or the scenarios where they need basic send/receive and wants to achieve that with as little code writing as possible.
+
+## Prerequisites
+1. [.NET Core](https://www.microsoft.com/net/core)
+2. An Azure subscription.
+3. [A ServiceBus namespace](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-create-namespace-portal) 
+4. [A ServiceBus queue](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dotnet-get-started-with-queues#2-create-a-queue-using-the-azure-portal)
+
+### Create a console application
+
+- Create a new .NET Core application. Check out [this link](https://docs.microsoft.com/en-us/dotnet/articles/core/getting-started) with help to create a new application on your operating system.
+
+### Add the ServiceBus client reference
+
+1. Add the following to your project.json, making sure that the solution references the `Microsoft.Azure.ServiceBus` project.
+
+    ```json
+    "Microsoft.Azure.ServiceBus": "0.0.7-preview"
```

Comment:
Is it possible to have this json without the version details? I'm afraid we will forget to update this version with every new release.